test/parallel_api/ranges/std_ranges_test.h

```diff
@@ -34,11 +34,15 @@ static_assert(ONEDPL_HAS_RANGE_ALGORITHMS >= 202409L);
 #include <ranges>
 #include <algorithm>
 #include <memory>
+#include <array>
 
 namespace test_std_ranges
 {
 
-inline constexpr std::size_t big_sz = (1<<25) + 10; //32M
+inline constexpr int big_sz = (1<<24) + 10; //16M
```

Comment:
It was 32M before.... What's a reason to changes it to 16M?
Don't we whether make test coverage worse?

Reply:
Testing 32M elements with device policies also is too much. There are no specializations for more than 16M elements.